.github/workflows/pylint.yml

```diff
@@ -0,0 +1,42 @@
+name: Pylint
+
+on: 
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
+
+    - name: Run Lint
+      working-directory: server
+      run: |
+        REPORT=$(pylint --exit-zero `ls -R|grep .py$|xargs`)
```

Comment:
Maybe would be nice to add a comment on top of that line on what is happening exactly for future references or for people who are interested. I know that '--exit-zero' is to treat all error as a warning but what is happening after that.